packages/orchestrator-ui-components/src/components/WFOSteps/WFOStepList/WFOStepList.tsx

```diff
@@ -0,0 +1,129 @@
+
+    return (
+        <>
+            <EuiFlexGroup css={stepListHeaderStyle}>
+                <EuiFlexGroup css={stepListContentStyle}>
+                    <EuiText css={stepListContentBoldTextStyle}>
+                        {t('steps')}
+                    </EuiText>
+                    <EuiText
+                        css={stepListContentAnchorStyle}
+                        onClick={openAllDetails}
+                    >
+                        {t('expandAll')}
+                    </EuiText>
+                </EuiFlexGroup>
+                <EuiFlexGroup
+                    justifyContent="flexEnd"
+                    direction="row"
+                    css={{ flexGrow: 0 }}
+                    gutterSize="s"
+                >
+                    <EuiButton
+                        onClick={(
+                            e: React.MouseEvent<
+                                HTMLButtonElement | HTMLElement,
+                                MouseEvent
+                            >,
+                        ) => {
+                            e.preventDefault();
+                            alert('TODO: Implement Show delta');
+                        }}
+                        iconSide="right"
+                        size="s"
+                        iconType="visVega"
+                    >
+                        {t('showDelta')}
+                    </EuiButton>
+                    <EuiButton
+                        onClick={(
+                            e: React.MouseEvent<
+                                HTMLButtonElement | HTMLElement,
+                                MouseEvent
+                            >,
+                        ) => {
+                            e.preventDefault();
+                            alert('TODO: Implement View options');
+                        }}
+                        iconType="eye"
+                        iconSide="right"
+                        size="s"
+                    >
+                        {t('viewOptions')}
+                    </EuiButton>
+                </EuiFlexGroup>
+            </EuiFlexGroup>
+            <>
+                {steps.map((step, index) => {
+                    const stepComponent = (
+                        <div key={`step-${index}`}>
+                            {index !== 0 && <div css={stepSpacerStyle} />}
+                            <WFOStep
+                                stepDetailIsOpen={
+                                    stepDetailStates.get(index) || false
+                                }
+                                toggleStepDetailIsOpen={toggleStepDetailIsOpen}
+                                step={step}
+                                stepIndex={index}
+                                startedAt={startedAt}
+                            />
+                        </div>
+                    );
+
+                    if (index > 0) {
+                        startedAt = step.executed;
```

Comment:
It looks like you are overriding an input prop of the steplist component here. You could do this inline without this reassignment (see comment above)